event-handlers/src/test/java/no/sikt/nva/nvi/events/evaluator/EvaluateNviCandidateWithSyntheticDataTest.java

```diff
@@ -53,8 +72,61 @@ void shouldParseDocumentWithManyContributorsWithinTimeOut(int numberOfForeignCon
             .getExpandedPublication();
 
     var expectedCreatorShares = numberOfNorwegianContributors + numberOfForeignContributors;
+    var candidate = evaluatePublication(publication);
+    assertThat(candidate.getCreatorShareCount()).isEqualTo(expectedCreatorShares);
+  }
+
+  @Test
+  void shouldIncludeAbstract() {
+    // TODO: Not implemented
```

Comment:
"Not implemented" in the Candidate, will rewrite these tests in the next PR when that is done.